src/composables/storageRef.ts

```diff
@@ -1,9 +1,16 @@
 import { Ref, ref, watch } from 'vue';
 import type { Migration, StorageKey } from '@/types';
-import { asyncPipe } from '@/utils';
 import { WalletStorage } from '@/lib/WalletStorage';
+import { asyncPipe } from '@/utils';
+import { SecureMobileStorage } from '@/lib/SecureMobileStorage';
+import { IS_MOBILE_APP } from '@/constants';
 
-interface ICreateStorageRefOptions<T> {
+export interface ICreateStorageRefOptions<T> {
+  /**
+   * Enable secure storage for the data.
+   * Mobile app only.
+   */
+  isSecure?: boolean;
```

Comment:
For me `isSecure` is like telling the code that the data we will be storing will be secure. If this options should enable different logic I suggest to rename this to something like `useSecureStorage`, `enableSecureStorage`, `forceSecureStorage`, `securityRequired`, etc...